### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

**Comment by Turbo87** [comment]
finally...

@trentmwillis @rwjblue this seems ready to go now

File: package.json [hunk 1]
```diff
@@ -22,11 +22,11 @@
...
+    "broccoli-concat": "^3.0.5",
+    "broccoli-funnel": "^1.0.5",
     "broccoli-jshint": "^1.1.2",
-    "broccoli-merge-trees": "^0.2.4",
-    "broccoli-sourcemap-concat": "^0.4.4",
-    "ember-cli": "1.13.13",
+    "broccoli-merge-trees": "^1.1.5",
+    "ember-cli": "^2.10.0",
     "ember-cli-release": "^1.0.0-beta.1",
     "ember-try": "0.2.2",
```
Comment (rwjblue) [inline package.json]: Can remove this

**Comment by Turbo87** [comment]
🍏

**Comment by rwjblue** [comment]
AWESOME, thank you!

**Pull request merged**